packages/hash/api/src/db/postgres/aggregation/createAggregation.ts

```diff
@@ -1,5 +1,5 @@
 import { Connection } from "../types";
-import { DbAggregation } from "../../adapter";
+import { DbAggregation, DbClient } from "../../adapter";
```

Comment:
Are we okay with having `DbClient` imported from here? I appreciate we can get some quality of life by not needing to keep types in sync, but we are creating a potentially annoying dependency graph. 
Just wanted to call it out in case you had some thoughts on this

Reply:
Interesting, I see what you're saying but I think this is fine because the methods we're implementing here are supposed to extend the `DbClient` interface - so I think this dependency here shouldn't actually introduce any _additional_ issues.